config/jobs/kubeflow/kubeflow-periodics.yaml

```diff
@@ -182,6 +182,38 @@ periodics:
   annotations:
     testgrid-dashboards: sig-big-data
     description: Periodic testing of Kubeflow examples
+  #******************************************************************************
+  # kubeflow/pipelines
+  #******************************************************************************
+- name: kubeflow-pipelines-periodic-functional-test
+  cluster: kubeflow
+  interval: 20m
+  decorate: true
+  labels:
+    preset-service-account: "true"
+  extra_refs:
+    - org: kubeflow
+      repo: pipelines
+      base_ref: master
+      workdir: true
+  spec:
+    containers:
+      - image: gcr.io/ml-pipeline-test/kfp-functional-test-base:latest
```

Comment:
best practice is to always use a tag, because our test only needs the static image

Reply:
done